tst/testinstall/grp/classic-G.tst

```diff
@@ -131,7 +131,7 @@ Error, <subfield> must be a prime or a finite field
 
 #
 gap> Omega(3,2);
-Omega(0,3,2)
+GO(0,3,2)
```

Comment:
Please also update `tst/testinstall/grp/classic-forms.tst` to verify the forms in these cases, i.e.,
```patch
--- a/tst/testinstall/grp/classic-forms.tst
+++ b/tst/testinstall/grp/classic-forms.tst
@@ -126,16 +126,9 @@ gap> for d in [3,5,7] do
 > od;
 gap> ForAll(grps, CheckGeneratorsSpecial);
 true
-
-# FIXME: forms are not implemented for odd d, even q
-# gap> ForAll(grps, CheckBilinearForm);
-# true
-# gap> ForAll(grps, CheckQuadraticForm);
-# true
-#
-gap> ForAll(Filtered(grps, g -> Characteristic(g)<>2), CheckBilinearForm);
+gap> ForAll(grps, CheckBilinearForm);
 true
-gap> ForAll(Filtered(grps, g -> Characteristic(g)<>2), CheckQuadraticForm);
+gap> ForAll(grps, CheckQuadraticForm);
 true
 gap> ForAll(grps, CheckSize);
 true
```